.github/workflows/deploy-pages-v1.yml

```diff
@@ -28,35 +30,44 @@ jobs:
       - name: Install dependencies
         run: pnpm install
 
-      - name: build
+      - name: Build v1
         env:
-          BASE_PATH: '/siwf'
-        run: |
-          pnpm run build
-      - name: structure
+          BASE_PATH_UI: /siwf/v1/ui
+          BASE_PATH_EXAMPLE: /siwf/v1/example
+        run: pnpm run build
+
+      - name: Structure v1
         run: |
           mkdir siwf
           touch siwf/.nojekyll
+          mkdir siwf/v1
```

Comment:
Deploying to BOTH /v1 and / now. So this will not break old stuff pointing at the unversioned path.